src/app/src/sensorUtils.js

```diff
@@ -0,0 +1,60 @@
+import axios from 'axios';
+import Papa from 'papaparse';
+
+import {
+    VARIABLES,
+    VARIABLE_DESCRIPTIONS,
+    VARIABLE_CODES,
+} from './constants';
+
+export function makeRiverGaugeRequest(id, isApiRequest) {
+    const url = isApiRequest
+        ? `https://waterservices.usgs.gov/nwis/iv/?format=json&sites=${id}`
+        : `https://nwis.waterdata.usgs.gov/nwis/qwdata/?site_no=${id}&agency_cd=USGS&inventory_output=retrieval&rdb_inventory_output=value&begin_date=2018-11-01&TZoutput=0&pm_cd_compare=Greater%20than&radio_parm_cds=all_parm_cds&qw_attributes=0&format=rdb&qw_sample_wide=wide&rdb_qw_attributes=0&date_format=YYYY-MM-DD&rdb_compression=value&submitted_form=brief_list`;
```

Comment:
Was the data of `begin_date=2018-11-01` chosen for a reason? Does this need to be dynamic?

Reply:
Good eye. Yes, that date is right before the last set of data as of today. It works going forward, assuming the dataset grows at a similar pace/doesn't balloon wildly. I don't know how we'd dynamically pick a date -- the API request doesn't have the option to grab the latest entries, AFAIK.

Reply:
I see. That makes sense to me.